redirects-www.conf

```diff
@@ -138,3 +138,5 @@ rewrite ^/general/library.shtml https://transition.fec.gov/general/library.shtml
 rewrite ^/links_files/Links.shtml http://classic.fec.gov/links_files/Links.shtml redirect;
 rewrite ^/eeo/NoFear.shtml /about/no-fear-act/ redirect;
 rewrite ^/info/espanol.shtml https://transition.fec.gov/info/espanol.shtml redirect;
+rewrite ^/finance/disclosure/candcmte_info.shtml /data redirect;
+rewrite ^/fecviewer/CandidateCommitteeDetail.do /data/?search=$arg_candidateCommitteeId redirect;
```

Comment:
Ah, clever!  I'd still be nice if we could figure out how to link directly but this at least gets something working now. :-)